aedes.js

```diff
@@ -347,5 +348,3 @@ function PublishState (broker, client, packet) {
   this.client = client
   this.packet = packet
 }
-
-function noop () {}
```

Comment:
The reason to avoid having a noop in a seperate file is the change of closure context. Please moving it back here.